src/main/java/Task/TaskType/Task.java

```diff
@@ -0,0 +1,42 @@
+package Task.TaskType;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "O" : "X"); //return tick or X symbols
+    }
+
+    public void markAsDone(){
+        // set isDone to be true
+        isDone = true;
+        // print the marked task
+        System.out.printf("    \n" +
+                "   Nice! I've marked this task as done:\n" +
+                "\t" + toString() +"\n");
+    }
```

Comment:
Might want to separate this into 2 classes since one class should have only one responsibility